### ISSUE TITLE
Cannot start

### ISSUE THREAD

**Issue by techmexdev** [xylafur/Bouncing-Square-Javascript#1]
A link to the program is missing,

**Comment by xylafur** [comment]
Yeah it's telling me that the submodule was not properly initialized with a .gitmodule and I'm too lazy right now to fix it.
